ci/bamboo/bootstrap-manylinux-prototype.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -o errexit
+set -o xtrace
+
+NUPIC_CORE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+
+#echo "NUPIC_CORE_DIR=${NUPIC_CORE_DIR}"
+
+DOCKER_IMAGE="quay.io/numenta/manylinux1_x86_64_centos6"
```

Comment:
I created the image without version, which defaults to "latest". This is also what the real manylinux project's image uses for version.